modules/ROOT/pages/7.4-release-notes.adoc

```diff
@@ -71,6 +71,23 @@ The {productname} {release-version} release includes an accompanying release of
 
 For information on the **<Premium plugin name 1>** plugin, see: xref:<plugincode>.adoc[<Premium plugin name 1>].
 
+=== Enhanced Code Editor
+// TINY-10561
+
+The {productname} {release-version} release includes an accompanying release of the **Enhanced Code Editor** premium plugin.
+
+**Enhanced Code Editor** includes the following fixes, additions, and improvements.
+
+==== Format button would lose focus after action while navigating through keyboard.
+// #TINY-11122
+
+Previously, after using the **Format code** button to format code, the focus would shift to the code editor wrapper instead of remaining on the **Format code** button.
+
+{productname} {release-version} implemented a fix that explicitly refocuses the **Format code** button after the formatting action.
+
+As a result, when pressing the **Format code** button, the focus now remains on the button as intended.
```

Comment:
```suggestion
Previously, after using the **Format code** button to format code, the focus would shift to the code editor wrapper instead of remaining on the **Format code** button. This behavior was inconsistent for users who prefer to navigate through the keyboard.

{productname} {release-version} implemented a fix that explicitly refocuses the **Format code** button after the formatting action. As a result, when pressing the **Format code** button, the focus now remains on the button as intended.
```